### PR TITLE
Feature/fight progress indicator

### PR DESCRIPTION
**Pull request by Yuyz0112**
- [x] add a ProgressBar component to indicate fight percentage

I implemented the progress bar by using svg path element. Because in this use case it is really simple, and avoid of using `position: absolute` or other more tricky solution to handle div align middle vertically.

### CODE REVIEW

**Comment by ryan-teamsolutionz** [comment]
Do we have a screenshot for this?

**Comment by Yuyz0112** [comment]
@ryan-teamsolutionz

**Comment by MartijnHols** [comment]
I don't like the bar on the next line as this reduces the amount of fights that can be shown within the viewport. Could you maybe show it on the same line, e.g. floating right of the fight name & wipe count?

Also I think it would look nicer if the white background was something like `rgba(0,0,0,0.6)` (might need fiddling).

**Comment by MartijnHols** [comment]
Also please update the changelog ;)

File: src/Main/FightSelecter.js [hunk 1]
```diff
@@ -92,6 +93,7 @@ class FightSelecter extends Component {
                       <Link to={makeAnalyzerUrl(report.code, fight.id)}>
                         <Fight {...fight} wipes={getWipeCount(report, fight)} />
                       </Link>
+                      <ProgressBar percentage={fight.kill ? 100 : (10000 - fight.fightPercentage) / 100}/>
```
Comment (MartijnHols) [inline src/Main/FightSelecter.js]: Please show this on the same line as the fight name & wiupe count.

**Comment by Yuyz0112** [comment]
@MartijnHols 
Saving the vertical space is a good point! I'm going to update the layout tonight.
I just create a codepen so you can test the color on it: https://codepen.io/aryu/pen/dREdEY

**Comment by Yuyz0112** [comment]
Changed request.

Now the progress bar float aside the kill/wipe span. Since the name of a fights are not in the same width, the progress bars could not be aligned left.
But in the fights of a same encounter it is easy to find the fight by the progress bar, and I think this is the scene needed fight progress.

**Comment by MartijnHols** [comment]
Thanks. I'll fiddle a bit with the layout as I'm not yet completely satisfied with the jumping around.

**Pull request merged**